tuf/api/metadata.py

```diff
@@ -487,8 +487,8 @@ def _type(self) -> str:
     def expires(self) -> datetime:
         """The metadata expiry date::
 
-        # Use 'datetime' module to e.g. expire in seven days from now
-        obj.expires = utcnow() + timedelta(days=7)
+        >>> # Use 'datetime' module to e.g. expire in seven days from now
+        >>> obj.expires = utcnow() + timedelta(days=7)
```

Comment:
The double colon (`::`) plus blank line makes Sphinx format this as code block. E.g. from our [RTD page](https://theupdateframework.readthedocs.io/en/latest/api/tuf.api.metadata.root.html#tuf.api.metadata.Root.expires):
![Screen Shot 2022-12-19 at 09 33 51](https://user-images.githubusercontent.com/589324/208382190-0f968a60-6dfc-427b-850b-d379ca1716f0.png)

What's the reason for this patch? 

(IIRC, the prompt is used by `doctest` to identify snippets to run, but we don't us that at the moment.)

Reply:
Thanks for review - I was not aware of the syntax with double colon. I think we can close the PR then.

Reply:
I also thought that indentation of the block would be needed (see last bullet point from our [docstrings guidelines](https://github.com/secure-systems-lab/code-style-guidelines/blob/master/python.md#docstrings), and the [sphinx docs they point to](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks))

But apparently it works without. 🤷